### ISSUE TITLE
compatibility with pyiron_atomistics broken

### ISSUE THREAD

**Issue by niklassiemer** [pyiron/pyiron_base#591]
We have broken pyiron_atomistics compatibility tests on our master since #565
```
======================================================================
ERROR: test_run (atomistics.master.test_murnaghan_non_modal.TestMurnaghan)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\pyiron_base\pyiron_atomistics\tests\atomistics\master\test_murnaghan_non_modal.py", line 68, in test_run
    self.project.wait_for_job(murn, interval_in_s=5, max_iterations=50)
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\project\generic.py", line 1271, in wait_for_job
    wait_for_job(
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\server\queuestatus.py", line 200, in wait_for_job
    raise ValueError("Maximum iterations reached, but the job was not finished.")
ValueError: Maximum iterations reached, but the job was not finished.

----------------------------------------------------------------------
```

**Comment by pmrv** [comment]
This somehow only happens windows build or did see that wrongly?

**Comment by jan-janssen** [comment]
Maybe we just have to increase the timeout

**Comment by niklassiemer** [comment]
Yes, it is only windoof. Is there a timeout set in this test? I did not see one... We do not run into the global timeout of 30 minutes we define in our tests. 
I also do not see a direct connection to this PR, however, since this PR is in the test always fails on our master.

**Comment by jan-janssen** [comment]
I disabled the interactive with class in https://github.com/pyiron/pyiron_base/pull/595 and then the compatibility tests work, while the unit tests fail as the interactive with class is not available. So there seems to be an issue with this class. @samwaseda Do you have any idea what could be causing this?

**Comment by jan-janssen** [comment]
By merging https://github.com/pyiron/pyiron_base/pull/595 the `with job.interactive.open()` functionality was disabled, because then the tests worked fine. I now opened a new pull request trying to re-enable the `with job.interactive.open()` functionality so we can debug it in https://github.com/pyiron/pyiron_base/pull/596

**Comment by samwaseda** [comment]
But `test_run` in `test_murnaghan_non_modal.py` doesn't even have `interactive`. I don't understand how this test and the PR could possibly be related.

**Comment by jan-janssen** [comment]
I do not understand it either, but when disabling it in https://github.com/pyiron/pyiron_base/pull/595 it worked again and by re-enabling it in https://github.com/pyiron/pyiron_base/pull/596 it failed again so there has to be some relation.

**Comment by samwaseda** [comment]
How can I see what failed?

**Comment by jan-janssen** [comment]
Funny this time it worked ... it really seems to be a random error ...

**Comment by liamhuber** [comment]
I don't know if it's related but over in #495 my OS X tests (and only them, even though they run fine on my macbook) had a worker job fail to get status finished 

Edited for cross ref

**Comment by jan-janssen** [comment]
As other tests are failing randomly as well like https://github.com/pyiron/pyiron_atomistics/actions/runs/1578841335/attempts/1 I have the feeling it might be related to an issue on the Github testing infrastructure. But it seems to appear only for tests which use a subprocess for execution.

**Comment by jan-janssen** [comment]
Maybe we can fix it with https://github.com/pyiron/pyiron_atomistics/pull/479

**Comment by jan-janssen** [comment]
So it seems like the pull request fixed the issue. I am closing it now - if it comes up again, feel free to open it again.

**Comment by niklassiemer** [comment]
Then I will gladly trigger a new release 😊

**Comment by liamhuber** [comment]
> So it seems like the pull request fixed the issue. I am closing it now - if it comes up again, feel free to open it again.

Can confirm. Merging in master resolved all issues over in #495
